README.md

```diff
@@ -23,27 +23,27 @@ The possibilities are endless.
 Now that you have not done anything it's time to build your application:
 
 ```
-
+  
 ```
 
 Yep. That's it. You should see the following output:
 
 ```
-
+   
 ```
 
 ### Deploying
 
 While you still have not done anything it's time to deploy your application. By running the following command you can deploy your application absolutely nowhere.
 
 ```
-
+    
```

Comment:
I feel like this can be optimize. Please remove the bloat, it is hideous to look at. :)